csrankings-s.csv

```diff
@@ -1808,6 +1808,7 @@ Stéphane Lafortune,University of Michigan,https://wiki.eecs.umich.edu/stephane/
 Stéphane Mallat,Ecole Normale Superieure,https://www.di.ens.fr/~mallat,g_YTmSgAAAAJ
 Stéphane S. Somé,University of Ottawa,http://www.site.uottawa.ca/~ssome,gZvSWooAAAAJ
 Stéphane Sotèg Somé,University of Ottawa,http://www.site.uottawa.ca/~ssome,gZvSWooAAAAJ
+Stephanie Balzer,Carnegie Mellon University,https://www.cs.cmu.edu/~balzers,xZXNqqYAAAAJ

```

Comment:
Research faculty not eligible for inclusion; please remove.

Reply:
Dr. Balzer is a tenure-track faculty member at the rank of Assistant Professor at the Computer Science Department, as of late 2023 or so.  This information can be found on the directory https://directory.andrew.cmu.edu/index.cgi?searchtype=guid&guid=B775EBF8-88AA-11E1-8001-00144F799A7A&activetab=basic, where it shows "Job Title According to HR: Assistant Professor".  Research faculty would be written as "Assistant Research Professor".

Reply:
The page you initially pointed me to says otherwise; I trust it is out of date and should be updated.

https://www.csd.cmu.edu/people/faculty/stephanie-balzer

<img width="891" alt="Screenshot 2024-05-26 at 9 57 25 AM" src="https://github.com/emeryberger/CSrankings/assets/1612723/9c8e5542-f921-40db-8514-15875a04d9b8">

Reply:
Thank you, Emery. Yes, that line does appear out of date. I will relay this information to have it updated.